README.md

```diff
@@ -178,3 +178,16 @@ differences.
 
 If you want to update the reference screenshot for a given example, you can grab
 those from the build artifacts as well and commit them to your branch.
+
+### Testing Locally
+
+Testing locally is possible, however pixel perfect results will differ from
+those on the CIs due to discrepencies in hardware, and driver (we use llvmpipe)
+versions.
+
+If you want to force the usage of LLVMPIPE to speed up local testing you
+may do so with the WGPUPY_WGPU_ADAPTER_NAME environment variable
```

Comment:
right?
```suggestion
If you want to force the usage of LLVMPIPE to enable local testing you
may do so with the WGPUPY_WGPU_ADAPTER_NAME environment variable
```

Reply:
And you need llvmpipe which is not trivial to install on non-Linux, right?

Reply:
BTW, why not just WGPUPY_ADAPTER_NAME?

Reply:
Honestly, I just tried to prefix the WGPU_ADAPTER_NAME, 

WGPUPY is a "wrapper" onto "WGPU", so we should have WGPUPY_WGPU_ADAPTER_NAME

and provides homage  to WGPU's WGPU_ADAPTER_NAME explicitly.

Reply:
i expaneded on the documentation warning against usage outside the WGPU-py test suite.